treeherder/webapp/api/resultset.py

```diff
@@ -151,3 +151,12 @@ def create(self, request, project, jm):
             jm.disconnect()
 
         return Response({"message": "well-formed JSON stored"})
+
+    @link()
+    @with_jobs
```

Comment:
this is fine with me.  But were we thinking of moving more to:
    with JobsModel(project) as jobs_model:
        return Response(jobs_model.get_resultset_status(pk))

that being said, all the other apis use ``@with_jobs`` so this is consistent.  I'm fine either way.  :)